lib/gui/lvgl/lv_conf.h

```diff
@@ -126,6 +130,28 @@ typedef void *lv_anim_user_data_t;
 #define LV_USE_SHADOW 0
 #endif
 
+#if LV_USE_SHADOW
+#define LV_SHADOW_CACHE_SIZE CONFIG_LVGL_SHADOW_CACHE_SIZE
+#endif
+
+#ifdef CONFIG_LVGL_BLEND_MODES
```

Comment:
This could be made more compact and understandable as:

    #define LV_USE_BLEND_MODES IS_ENABLED(CONFIG_LVGL_BLEND_MODES)

Multiple instances.  I won't insist on it because existing code would have to change and this file is horrible anyway.

Reply:
Thanks for the suggestion @pabigot. I think that can be addressed in the planned PR where re-organization of Kconfig will be handled.